File: src/main/java/com/starshootercity/abilities/EnderParticles.java
package com.starshootercity.abilities;

import net.kyori.adventure.key.Key;
import org.bukkit.Particle;
import org.jetbrains.annotations.NotNull;

public class EnderParticles implements ParticleAbility {
    @Override
    public @NotNull Key getKey() {
        return Key.key("origins:ender_particles");
    }

    @Override
    public Particle getParticle() {
        return Particle.PORTAL;
    }

    @Override
    public int getFrequency() {
        return 4;
    }
}

File: core/src/main/java/com/starshootercity/abilities/EnderParticles.kt
package com.starshootercity.abilities

import net.kyori.adventure.key.Key
import org.bukkit.Particle

class EnderParticles : ParticleAbility {
    override fun getKey(): Key {
        return Key.key("origins:ender_particles")
    }

    override fun getParticle(): Particle {
        return Particle.PORTAL
    }

    override fun getFrequency(): Int {
        return 4
    }
}
